### PR TITLE
PKI.NCALayer v1.3

### PR DESCRIPTION
**Pull request by Kesantielu**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/49989)

### CODE REVIEW

File: manifests/p/PKI/NCALayer/1.3/PKI.NCALayer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=NLSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.1.0.schema.json
+
+PackageIdentifier: PKI.NCALayer
+PackageVersion: "1.3"
+PackageLocale: en-US
+Publisher: National Certification Authority of the Republic of Kazakhstan
```
Comment (ItzLevvie) [inline manifests/p/PKI/NCALayer/1.3/PKI.NCALayer.locale.en-US.yaml:7]:
```suggestion
Publisher: Национальный удостоверяющий центр РК
```
shows up as `Национальный удостоверяющий центр РК` on en-US devices.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by Kesantielu** [comment]
> Hello @Kesantielu, When processing this package, the URL to the installer returned a 404 error or was determined to be invalid.
> 
> Please update the URL and hash at your convenience and resubmit your PR.

I don't know why the autocheck fails.

```
curl -I https://ncl.pki.gov.kz/images/NCALayer/NCALayerInstall_x86.exe
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 04 Mar 2022 07:48:50 GMT
Content-Type: application/octet-stream
Content-Length: 54828392
Last-Modified: Fri, 21 Jan 2022 06:57:04 GMT
Connection: keep-alive

curl -I https://ncl.pki.gov.kz/images/NCALayer/NCALayerInstall_x64.exe
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 04 Mar 2022 07:48:55 GMT
Content-Type: application/octet-stream
Content-Length: 58833312
Last-Modified: Fri, 21 Jan 2022 06:57:03 GMT
Connection: keep-alive
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
> Commenter does not have sufficient privileges for PR 49989 in repo microsoft/winget-pkgs

You can close and re-open the PR to re-run the pipelines.

> I don't know why the autocheck fails.

Maybe the URL might be blocked in Microsoft Azure or it's downloading very slowly which is why it might be erroring out.

**Review by ImJoakim — APPROVED** [review]

**Pull request merged**